ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.16.yaml

```diff
@@ -324,6 +324,7 @@ tests:
     cluster_profile: nutanix-qe-zone
     env:
       CLUSTER_OS_IMAGE: https://rhcos.mirror.openshift.com/art/storage/prod/streams/4.16-9.4/builds/416.94.202405171948-0/x86_64/rhcos-416.94.202405171948-0-nutanix.x86_64.qcow2
+      ENABLE_REBOOT_CHECK: "true"
```

Comment:
suggest to append key word "reboot-nodes" on the job name to reflect the test

Reply:
Fixed, pls take a look.